packages/@ama-sdk/core/src/plugins/session-id/session-id.request.ts

```diff
@@ -32,27 +33,38 @@ export class SessionIdRequest implements RequestPlugin {
   /**
    * The session ID used to track API calls
    */
-  public sessionId: string;
+  public sessionId!: string;
```

Comment:
It is now really potentially `undefined`:

```suggestion
  public sessionId?: string;
```